content/applications/inventory_and_mrp/inventory/product_management/inventory_valuation/valuation_by_lots.rst

```diff
@@ -0,0 +1,212 @@
+.. figure:: valuation_by_lots/lip-gloss.png
+   :align: center
+   :alt: Show Cost on the product form.
+
+   As a result, the product form displays an average cost of $15 in the **Cost** field.
+
+.. _inventory/product_management/valuation-cost-new:
+
+Create new lot/serial number
+----------------------------
+
+Creating a new lot/serial number through an :doc:`inventory adjustment
+<../../warehouses_storage/inventory_management/count_products>` assigns the same value as the cost
+on the product form.
+
+To make an inventory adjustment and assign a lot number, go to :menuselection:`Inventory app -->
+Operations --> Adjustments: Physical Inventory`, then, click :guilabel:`New`.
+
+In the new inventory adjustment line, set the :guilabel:`Product`,
+create the :guilabel:`Lot/Serial Number`, set the :guilabel:`Counted Quantity`, and click
+:icon:`fa-floppy-o` :guilabel:`Apply`.
+
+To view the valuation layer, go to :menuselection:`Inventory app --> Reporting --> Valuation`. The
+:guilabel:`Total Value` per unit matches the cost on the product form.
+
+.. example::
+   Continuing the example in the table above, when the product cost is `$15`, the valuation for a
+   newly created `LOT3` is also be `$15`.
+
+   .. image:: valuation_by_lots/create-new.png
+      :align: center
+      :alt: Show inventory adjustment valuation.
+
+.. _inventory/product_management/valuation-cost-existing:
+
+Existing lot/serial number
+--------------------------
+
+When adjusting the quantity of an existing lot/serial number, the value is based on the most recent
+valuation layer that specific lot/serial number.
+
+.. example::
+   Continuing the example in the table above, LOT 1's value is $10.
+
+   So, when the quantity is updated from `1.00` to `2.00`, the additional quantity is also valued at
+   `$10`, reflecting the latest valuation layer for `LOT 1`.
+
+   .. figure:: valuation_by_lots/existing.png
+      :align: center
+      :alt: Show valuation of LOT 1 getting updated.
+
+      The inventory adjustment (top line) is valued the same as LOT 1 (bottom line).
+
+.. _inventory/product_management/view-valuation:
+
+View valuation
+==============
+
+To find the average cost of a specific lot/serial number, go to :menuselection:`Inventory app -->
+Products --> Lots/Serial Numbers` and select the desired record.
+
+Both the :guilabel:`Cost` and :guilabel:`Average Cost` fields show a unit's average cost. The
+:guilabel:`Total Value` reflects the total on-hand value for that lot/serial number.
+
+.. figure:: valuation_by_lots/lot.png
+   :align: center
+   :alt: Show cost of the lot/serial number.
+
+   Lot form, displaying **Cost** field. The **Valuation** smart button is in the top-right.
+
+Valuation layers of a lot/serial number can be viewed through the :ref:`valuation report
+<inventory/product_management/valuation-report>` or by clicking the lot/serial number's
+:guilabel:`Valuation` smart button. These detailed, line-by-line records can help determine how each
```

Comment:
I am not seeing any 'Valuation' smart button on the lot/serial number form in the Master runbot I am using for this review. This should be double-checked during your revisions.

![Screenshot 2024-09-30 105936](https://github.com/user-attachments/assets/3f01d956-96d6-4b3c-9396-344e02c9560e)

Reply:
Thanks for sharing the URL in your screenshot, so I can diagnose the issue (:

In the product category, please make sure the costing method is set to FIFO or AVCO.
![image](https://github.com/user-attachments/assets/d57e620c-fdd4-4a32-b010-e10f545c2723)

![image](https://github.com/user-attachments/assets/26505c72-dd8d-43bd-8d10-a2303ef69d1e)


While I do have the instructions in the doc to make sure to set the costing method correctly, please let me know if you have suggestions on how to make it clearer (: